### PR TITLE
Add Made with Prisma badge

### PR DESCRIPTION
**Pull request by petradonka**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -57,6 +57,19 @@ You can learn more about Prisma and its rich ecosystem and community with these
 
 </details>
 
+## Badges
+
+[![Made with Prisma](http://made-with.prisma.io/dark.svg)](https://prisma.io) [![Made with Prisma](http://made-with.prisma.io/indigo.svg)](https://prisma.io)
+
+Built something awesome with Prisma? 🌟 Show it off with these [badges](https://github.com/prisma/presskit?tab=readme-ov-file#badges), perfect for your readme or website.
```
Comment (nikolasburk) [inline profile/README.md:64]: I see README typically spelled being all-caps:

```suggestion
Built something awesome with Prisma? 🌟 Show it off with these [badges](https://github.com/prisma/presskit?tab=readme-ov-file#badges), perfect for your README or website.
```

File: profile/README.md [hunk 2]
```diff
@@ -57,6 +57,19 @@ You can learn more about Prisma and its rich ecosystem and community with these
...
+
+```
```
Comment (nikolasburk) [inline profile/README.md:66]: we could use a markdown annotation for syntax highlighting here:

````suggestion
```md
````

File: profile/README.md [hunk 3]
```diff
@@ -57,6 +57,19 @@ You can learn more about Prisma and its rich ecosystem and community with these
...
+[![Made with Prisma](http://made-with.prisma.io/dark.svg)](https://prisma.io)
+```
+
+```
```
Comment (nikolasburk) [inline profile/README.md:70]: we could use a markdown annotation for syntax highlighting here:

````suggestion
```md
````

**Review by nikolasburk — APPROVED** [review]
I've left some small suggestions for consideration, none of these blockers to merge 👍

File: profile/README.md (hunk 2 above)
Comment (petradonka) [reply profile/README.md:66]: Thanks for the suggestion: I'll add the same comment as on the other repo. The syntax highlighting didn't show up for me as any difference on presskit, so I'll just merge this in. Very happy to add this later on if it turns out maybe I missed something.

**Pull request merged**